.travis.yml

```diff
@@ -13,15 +13,15 @@ services:
 env:
   global:
     - BASE_IMAGE_NAME="${DOCKER_ORG}/geospaas:2.3.3-slim"
-    - IMAGE_NAME="${DOCKER_ORG}/geospaas_processing_worker"
+    - IMAGE_NAME_WORKER="${DOCKER_ORG}/geospaas_processing_worker"
+    - IMAGE_NAME_CLI="${DOCKER_ORG}/geospaas_processing_cli"
     - IDF_CONVERTER_VERSION=0.0.124
+    - GEOSPAAS_DB_HOST='db'
+    - GEOSPAAS_DB_USER='test'
```

Comment:
you can put those variables back in the unit test stage

Reply:
Done